index.html

```diff
@@ -11,36 +11,44 @@
     <main>
         <header>
           <h1>
-            My Personal Profile
+            My Professional Portfolio - Jeffery Ray Russell
           </h1>
           <p></p>
         </header>
+
         <nav>
           <ul>
-            <li>Articles</li>
-            <li>About (coming soon!)</li>
-            <li><a href="http://twitter.com/brookr" target="_blank">Twitter</a></li>
-            <li><a href="http://github.com/brookr" target="_blank">Github</a></li>
+            <li>Portolio Projects</li>
+            <li>About Me (coming soon!)</li>
+            <li><a href="http://twitter.com/jefferyrayruss2" target="_blank">Twitter</a></li>
+            <li><a href="https://github.com/jefferyrayrussell" target="_blank">Github</a></li>
+            <li><a href="http://facebook.com/jeff.russell.5" target="_blank">Facebook</a></li>
           </ul>
         </nav>
-        <section id="articles">
-          <article class="template" data-category="cat">
+
+        <section id="portfolio-projects">
+          <article class="template" data-category="category">
+
             <header>
-              <h1>Title</h1>
-              <div class="byline">
+REWORK THE HEADER SECTION
+              <h1>Project Title</h1>
+              <div class="project-details">
                 By <address><a href="">Author Name</a></address>
                 published <time pubdate datetime="2000-01-01">Publish Time</time>
               </div>
+
             </header>
-            <section class="article-body"></section>
+            <section class="project-summary"></section>
+
           </article>
         </section>
       </div>
+
     </main>
     <footer>
     </footer>
       <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
-      <script src="scripts/blogArticles.js"></script>
-      <script src="scripts/article.js"></script
+      <script src="scripts/portfolioProjects.js"></script>
+      <script src="scripts/app.js"></script
```

Comment:
These script tags should not be indented further than the footer.  And this last one is missing a bracket.